src/data/auroradb.ts

```diff
@@ -0,0 +1,255 @@
+    fromCommonName: string;
+    toCommonName: string;
+    journeyPatternId: string;
+    order: string;
+}
+export interface RawJourneyPatternStops {
+    OrderedStopPoints: {
+        StopPointRef: string;
+        CommonName: string;
+    }[];
+}
+
+export interface RawJourneyPattern {
+    JourneyPattern: RawJourneyPatternStops[];
+}
+
+interface NaptanInfo {
+    commonName: string;
+    naptanCode: string;
+    atcoCode: string;
+    nptgLocalityCode: string;
+    localityName: string;
+    parentLocalityName: string;
+    indicator: string;
+    street: string;
+}
+
+interface NaptanAtcoCodes {
+    naptanCode: string;
+    atcoCode: string;
+}
+
+export interface Stop {
+    stopName: string;
+    naptanCode: string;
+    atcoCode: string;
+    localityCode: string;
+    localityName: string;
+    parentLocalityName: string;
+    indicator?: string;
+    street?: string;
+    qualifierName?: string;
+}
+
+export interface StopIdentifiers {
+    naptanCode: string | null;
+    atcoCode: string;
+}
+
+export interface Service {
+    serviceDescription: string;
+    operatorShortName: string;
+    journeyPatterns: JourneyPattern[];
+}
+
+export interface RawService {
+    serviceDescription: string;
+    operatorShortName: string;
+    journeyPatterns: RawJourneyPattern[];
+}
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+const getAuroraDBClient = () => {
+    let client = null;
+
+    if (process.env.NODE_ENV === 'development') {
+        client = createPool({
+            host: 'localhost',
+            user: 'fdbt_site',
+            password: 'password',
+            database: 'fdbt',
+            waitForConnections: true,
+            connectionLimit: 10,
+            queueLimit: 0,
+        });
+    } else {
+        const ssm = new AWS.SSM();
+        client = createPool({
+            host: process.env.RDS_HOST,
+            user: ssm.getParameter({ Name: 'fdbt-rds-site-username', WithDecryption: true }).toString(),
+            password: ssm.getParameter({ Name: 'fdbt-rds-site-password', WithDecryption: true }).toString(),
+            database: 'fdbt',
+            waitForConnections: true,
+            connectionLimit: 10,
+            queueLimit: 0,
+        });
+    }
+
+    return client;
+};
+
+export const convertDateFormat = (startDate: string): string => {
+    return dateFormat(startDate, 'dd/mm/yyyy');
+};
+
+const connection = getAuroraDBClient();
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const executeQuery = async (query: string, values: any[]) => {
+    const [rows] = await connection.execute(query, values);
+    return rows;
+};
+
+export const getServicesByNocCode = async (nocCode: string): Promise<ServiceType[]> => {
+    try {
+        const queryInput = 'SELECT * FROM `tndsService` WHERE `nocCode` = ?';
+
+        const queryResult = await executeQuery(queryInput, [nocCode]);
+
+        const resultRow = JSON.parse(JSON.stringify(queryResult));
```

Comment:
Could we move this parse and stringify into the `executeQuery` method to save having to do it in every method?